### PR TITLE
added support for ExposeExceptions

### PR DESCRIPTION
**Pull request by mahald**
added support for ExposeExceptions as discussed in #279

### CODE REVIEW

**Comment by mahald** [comment]
Ok here the PR with copy the config variable to ExecutionResult. See #279

File: src/GraphQL/Execution/ExecutionOptions.cs [hunk 1]
```diff
@@ -27,5 +27,7 @@ public class ExecutionOptions
         public readonly IList<IDocumentExecutionListener> Listeners = new List<IDocumentExecutionListener>();
 
         public IFieldNameConverter FieldNameConverter { get; set; } = new CamelCaseFieldNameConverter();
+
+        public bool ExposeExceptions = false;
```
Comment (joemcbride) [inline src/GraphQL/Execution/ExecutionOptions.cs]: Mind switching this to a property?

**Comment by mrdezzods** [comment]
@joemcbride  Can you please make a new release including this commit ?

Thanks

**Comment by joemcbride** [comment]
@mrdezzods This is released in [0.17.3](https://github.com/graphql-dotnet/graphql-dotnet/releases/tag/v0.17.3).

**Comment by mrdezzods** [comment]
@joemcbride: now eagerly waiting for #415 to be fixed :)

**Pull request merged**